frontend/src/pages/LoginPage.vue

```diff
@@ -57,12 +82,24 @@ export default {
       }
       return false;
     },
-    logIn() {
+    async logIn() {
+      this.loading = true;
       if(this.formValidation()) {
-        //This will be replaced with call to http
-        console.log("Username:"+this.username.value);
-        console.log("Password:"+this.password.value);
+        try {
+          await this.$store.dispatch('logIn', {username: this.username.value, password: this.password.value});
+          this.$router.push(`/user/${this.username.value}`);
+        } catch(err) {
+          this.resultTitle = 'Error';
+          this.resultType = 'error';
+          this.resultMessage = err.message;
```

Comment:
This is going to make much more sense (the error message) once we deal with sending sensible error messages from the backend, right?

Reply:
Because currently it's "Error logging in"

Reply:
This error is mainly for network errors or something like that. Errors for invalid passwords or usernames will be displayed on form inputs. So yes, this is going to make more sense